lib/template.js

```diff
@@ -15,6 +19,41 @@ const template = (string, obj) => {
     }
     return result
   })
+  if (customReplacers) {
+    customReplacers.forEach(({ search, replace }) => {
+      str = str.replace(search, replace)
+    })
+  }
+  return str
+}
+
+function validateReplacers({ app, context, replacers }) {
+  return replacers
+    .map(replacer => {
+      try {
+        if (replacer.regex) {
+          return {
+            search: regexParser(replacer.regex),
```

Comment:
cheeky *beep* :laughing: I spent an hour thinking my schema addition was borked when in reality you modified the schema. :rage: At myself for now reading the code my closely before deleting it :laughing: